index.md

```diff
@@ -722,38 +758,18 @@ Let's put the sixth item in `wikiCatNameTables` into an object and see what we g
 
 ```r
 famousCats <- wikiCatNameTables[[5]]
```

Comment:
Looks like this index should be `2` instead of `5`

Reply:
Yeah, that's what I first thought, but after I wondered if the table you were aiming to retrieve was somewhere else https://en.wikipedia.org/wiki/List_of_fictional_felines. Maybe not, I will do what's I think is best. The other issue I had was that `rvest` returns an object of class `tbl` (along with 2 other classes) so as `tibble` is installed on my machine, the print is the one of a `tibble`, which might be confusing for beginners. I guess I can use a  docker container to overcome this.

Reply:
@sckott I ended up using the wiki page mentioned above, I thought it was the simplest way forward. I can always use the former page and use 2 instead of 5 as suggested, but I would need to edit the text consistently!

Reply:
LGTM